Source/Example Measurements/NI-Digital SPI/README.md

```diff
@@ -0,0 +1,26 @@
+## NI-Digital SPI
+
+This is a MeasurementLink example that tests an SPI device using an NI Digital Pattern instrument.
+
+### Features
+
+- Uses the `nidigital` package to access the NI-Digital Pattern Driver from LabVIEW
+- Pin-aware, supporting one session, multiple pins, and multiple selected sites
+- Includes project files for Digital Pattern Editor, InstrumentStudio, MeasurementLink UI Editor
```

Comment:
The MeasurementLink UI Editor project should have been removed.

Reply:
Fixed as suggested. Forgot to update the readme when removing it.